docs/src/man/structure.md

```diff
@@ -244,6 +232,210 @@ julia> rad2deg(psiangle(struc['A'][50], struc['A'][51]))
+| :----------------------------- | :-------------------------------------------------------------------------------------------------------------------- |
+| `pdb_dir::AbstractString=pwd()`| The directory to which the PDB file is downloaded                                                                     |
+| `file_format::Type=PDB`        | The format of the PDB file. Options are PDB, PDBXML, MMCIF or MMTF                                                    |
+| `obsolete::Bool=false`         | If set `true`, the PDB file is downloaded into the auto-generated "obsolete" directory inside the specified `pdb_dir` |
+| `overwrite::Bool=false`        | If set `true`, overwrites the PDB file if exists in `pdb_dir`; by default skips downloading the PDB file              |
+| `ba_number::Integer=0`         | If set > 0, downloads the respective biological assembly; by default downloads the PDB file                           |
+
+
+## Reading PDB files
+
+- To parse a existing PDB file into a Structure-Model-Chain-Residue-Atom framework:
+
+    ```julia
+    julia> struc = read("/path/to/pdb/file.pdb", PDB)
+    Bio.Structure.ProteinStructure
+    Name                        -  1EN2.pdb
+    Number of models            -  1
+    Chain(s)                    -  A
+    Number of residues          -  85
+    Number of point mutations   -  5
+    Number of other molecules   -  5
+    Number of water molecules   -  76
+    Number of atoms             -  614
+    Number of hydrogens         -  0
+    Number of disordered atoms  -  27
+    ```
+
+    Various options can be set through optional keyword arguments when parsing a PDB file as follows:
+
+| Keyword Argument                             | Description                                                                     |
+| :------------------------------------------- | :------------------------------------------------------------------------------ |
+| `structure_name::AbstractString="$pdbid.pdb"`| The name of the PDB Structure read. Defaults to "< PDBID >.pdb"                 |
+| `remove_disorder::Bool=false`                | If set true, then disordered atoms wont be parsed                               |
+| `read_std_atoms::Bool=true`                  | If set false, then standard ATOM records wont be parsed                         |
+| `read_het_atoms::Bool=true`                  | If set false, then HETATOM records wont be parsed                               |
+
+- To parse a PDB file by specifying the PDB ID and PDB directory into a Structure-Model-Chain-Residue-Atom framework:
+
+    ```julia
+    struc = readpdb("1EN2", pdb_dir="/path/to/pdb/directory")
+    ```
+
+    **Note:** This requires the PDB file name to be uppercase PDB ID. Example : "1EN2.pdb"
+
+    The function `readpdb` provides an uniform way to download and read PDB files. For example:
```

Comment:
I'm not sure lines 309-337 are required. Perhaps we could just say "The function `readpdb` provides an uniform way to download and read PDB files, for example `readpdb("1EN2",pdb_dir="/path/to/pdb/directory")`. The same keyword arguments are taken as `read` above, plus `pdb_dir` and `ba_number`."